src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -117,11 +130,6 @@ export default function FormNav(props) {
 }
 
 FormNav.defaultProps = {
-  formConfig: {
-    customText: {
-      reviewPageTitle: '',
-    },
-  },
```

Comment:
Removed because of lint warning - `formConfig` is required, and therefore doesn't need a default definition